1-create-unprivileged-user.sh

```diff
@@ -19,17 +19,32 @@ data:
   htpasswd: ${htpwd_encoded}
 EOF
 
-kubectl patch oauth cluster --type='json' -p '[
-  {
-    "op":"add","path":"/spec/identityProviders/1",
-    "value":{
-      "name":"htpasswd",
-      "mappingMethod":"claim",
-      "type":"HTPasswd",
-      "htpasswd":
-        {"fileData":
-          {"name":"htpass-secret"}
-        }
+# if there are no indentityProviders
+if [[ $(kubectl get oauth cluster -o=jsonpath='{.spec.identityProviders[*].name}' | wc -c) -eq 0 ]]; then
```

Comment:
on RHPDS cluster which has a list of pre-created users no extra users would be created, right?

Reply:
On an RHPDS cluster, the [else if](https://github.com/redhat-developer/devspaces-demo/blob/ab2131de9dd81d11df16362fdf26d3b3fbb161cf/1-create-unprivileged-user.sh#L35) block would run and would add a new identityProvider alongside the already-existing one, so the pre-created users are still able to login, as well as the new extra users